### PR TITLE
Documentation of coregctpopup

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function coregctpopup. Description and inputs in need of completion.

### CODE REVIEW

File: ea_hyperlink_label.m [hunk 1]
```diff
@@ -1,4 +1,20 @@
 function ea_hyperlink_label(label, url, position)
+%
```
Comment (Daniel-Tojal) [inline ea_hyperlink_label.m]: Please provide a description of the function

File: ea_hyperlink_label.m [hunk 2]
```diff
@@ -1,4 +1,20 @@
 function ea_hyperlink_label(label, url, position)
+% 
+%
+% USAGE:
+%
+%    ea_hyperlink_label(label, url, position)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_hyperlink_label.m]: Please provide a description of the

File: ea_init_coregctpopup.m [hunk 3]
```diff
@@ -1,4 +1,19 @@
 function ea_init_coregctpopup(handles,options,handlestring)
+%
```
Comment (Daniel-Tojal) [inline ea_init_coregctpopup.m]: Please provide a description of the function

File: ea_init_coregctpopup.m [hunk 4]
```diff
@@ -1,4 +1,19 @@
 function ea_init_coregctpopup(handles,options,handlestring)
+% 
+%
+% USAGE:
+%
+%    [options, presentfiles] = ea_assignpretra(options,allowgz)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_init_coregctpopup.m]: Please provide a description of the inputs

File: ea_init_coregctpopup.m [hunk 5]
```diff
@@ -15,7 +30,7 @@ function ea_init_coregctpopup(handles,options,handlestring)
             cdc{cnt}=thisndc;
             coregctmethod{cnt}=methodf;
             if strcmp(cdc{cnt},eval([options.prefs.ctcoreg.default,'(','''prompt''',')']))
-                defentry=cnt;
+                defentry=cnt;q
```
Comment (laurentheirendt) [inline ea_init_coregctpopup.m]: @Daniel-Tojal, I think this is a typo.

Comment (Daniel-Tojal) [reply ea_init_coregctpopup.m]: it is, thank you

File: ea_init_coregctpopup.m (hunk 3 above)
Comment (ningfei) [reply ea_init_coregctpopup.m]:
```
% Initialize CT coregistration methods popupmenu.
%
% INPUTS:
%    handles: LEAD GUI handle
%    refine: deprecated.
```

**Comment by laurentheirendt** [comment]
Overlap with #260. May you remove the doc for the hyperlink function?

**Pull request closed without merging**